source/docs/contributing/frc-docs/style-guide.rst

```diff
@@ -3,7 +3,7 @@
 Style Guide
 ===========
 
-This document contains the various RST/Sphinx specific guidelines for the frc-docs project. For guidelines related to the various WPILib code projects, see `the WPILib GitHub <https://github.com/wpilibsuite/styleguide>`__
+This document contains the various RST/Sphinx specific guidelines for the frc-docs project. For guidelines related to the various WPILib code projects, see [the WPILib GitHub](https://github.com/wpilibsuite/styleguide)
 
 Filenames
```

Comment:
Should new files keep using .rst, or change?

Reply:
I'm open to either (keep files as .rst or change to .rd).
Changing the file extension will break all open PRs so I'm more weary of it